app/Services/QuizAttemptService.php

```diff
@@ -25,36 +25,53 @@ public function startQuiz($quizId, $data)
         return $quizAttempt;
     }
 
-    public function submitAnswers($quizAttemptId, $data)
-{
-    $quizAttempt = QuizAttempt::query()->findOrFail($quizAttemptId);
-    $score = 0;
+    public function countUserAttempts($quizId)
```

Comment:
you can do this logic in the start quiz service before you can check how many attempts that user has no need for seperate function